src/daemons/MetaDaemon.cpp

```diff
@@ -306,7 +306,9 @@ void signalHandler(int sig) {
             }
             {
                 auto gJobMgr = nebula::meta::JobManager::getInstance();
-                gJobMgr->shutDown();
+                if (gJobMgr) {
```

Comment:
Here we also need to determine if gJobMgr has called the init function.

Reply:
good catch